docs/coreConcepts/LitActionsAndPKPs/intro.md

```diff
@@ -12,9 +24,9 @@ We're building this at Lit: The smart contracts are Lit Actions and the key pair
 
 ## Programmable Key Pairs (PKPs)
 
-PKPs are public/private key pairs generated by the Lit network in a process called Distributed Key Generation. Each node custodies a *share* of the underlying private key, meaning the key **never exists in its entirety**. Currently, users can mint a PKP in the form of an ERC-721 NFT. The owner of the NFT becomes the sole controller of the underlying private key, their wallet signature becoming the method of authorization for the Lit nodes.
+PKPs are public/private key pairs generated by the Lit network in a process called Distributed Key Generation. Each node custodies a *share* of the underlying private key, meaning the key **never exists in its entirety**. Currently, users can mint a PKP in the form of an ERC-721 NFT. Only the owner of the NFT has the ability to query the Lit network to combine the underlying private key shares, their wallet signature standing as the method of authorization for the Lit nodes.
```

Comment:
Same comment as above -> it is who has access to sign with the PKP has the ability to combine the underlying private key shares.